datadog_checks_base/datadog_checks/base/checks/win/wmi/sampler.py

```diff
@@ -390,34 +390,102 @@ def _format_filter(filters, and_props=[]):
         """
         Transform filters to a comprehensive WQL `WHERE` clause.
 
-        Specifying more than 1 filter defaults to an `OR` operator in the `WHERE` clause.
+        Specifying more than 1 filter defaults to an `OR` operator in the `WHERE` clause unless specified otherwise.
```

Comment:
> Specifying more than 1 filter defaults to an `OR` operator in the `WHERE` clause unless specified otherwise.

How can I specify a different operator to be used between items of `fitlers`? The doc below only shows how to specify operator to be used between values of a single property.